packages/prepo-utils/src/validateStringToBN.ts

```diff
@@ -1,5 +1,7 @@
 export const validateStringToBN = (input: string): boolean => {
   // empty string is treated as 0
   if (input === '') return true
+  // valid strings can't have spaces
+  if (input !== input.trim()) return false
```

Comment:
simpler alternative solution IMO, but not blocker

```suggestion
  if (input.includes(' ')) return false
```

Reply:
Disregarding this because `trim` removes more than just spaces. It removes any white space character and line breaks. I don't think that you can enter those in the input but it's good to be defensive.